client/state/products-list/schema.js

```diff
@@ -20,7 +20,7 @@ export const productsListSchema = {
 				product_name: { type: 'string' },
 				product_slug: { type: 'string' },
 				description: { type: 'string' },
-				cost: { type: [ 'string', 'null' ] },
+				cost: { type: [ 'integer', 'null' ] },
```

Comment:
This assumption might come back and bit us in the behind - there's no guarantee that for whatever reason the cost will not have something after the decimal point (on the backend it's stored as `decimal(10,2)`).

Reply:
Can probably use number or float, fwiw.

Reply:
After thinking about it - the previous approach also assumes an integer, so I guess it does not matter for this PR. If we wanted to fix this, we'd probably need to fix other places as well anyway.